app/views/application/_survey_widget.html.erb

```diff
@@ -0,0 +1,19 @@
+<%#
+Survey PR:
+	https://github.com/sciencehistory/scihist_digicoll/pull/1917
+Survey settings:
+	https://sciencehistory.surveysparrow.com/survey/330435/results/questions
+	https://sciencehistory.surveysparrow.com/survey/330435/channels/2900170/embed/edit
+%>
+<% if current_user%>
+	<%# <div class="alert alert-primary">Survey hidden because you are logged in.</div> %>
+<% elsif ScihistDigicoll::Env.lookup!(:show_survey_sparrow) == "true" %>
+	<div id="ss_survey_widget" />
+	<script id="SS_SCRIPT"><%# 
+		This script tag is only here to avoid an error in the the canned JS we need to put on the site.
+		For the actual JS code, see the GitHub PR above.%></script>
+<% else %>
+	<%#
+		<div class="alert alert-primary">Survey hidden because show_survey_sparrow is not turned on</div>
+	%>
+<% end %>
```

Comment:
can you remove the commented-out stuff? 

So it could just be `if !current_user && ScihistDigicoll::Env.lookup!(:show_survey_sparrow) == "true"`...  I'm not personally convinced we need to hide this for logged in users... can't all 6 of us just close the survey once without taking it? But if you want to, also fine.

Reply:
Valid points, but I am leaving the code as is; my arguements are a) this is temporary code and b) there's a real chance we may be debugging it again soon. My guess is that having all the extra comments might just save us a bit of work later on.